wireapp/wire-android
# Bugfix : Linear on boarding m3

## What's new in this PR?

### Issues

Deeplink LOG IN WITH SSO does not respond sometimes.

https://wearezeta.atlassian.net/browse/AN-6684

### Causes

The flow for milestone 3 was broken when the endpoint `/sso/settings` returns null value .

`    {"default_sso_code":null}`

### Solutions

add this test case   `case Some("") => SSONotFound`

### Testing

-  Open a deeplink to staging backend : wire://access/?config=https://s3-eu-west-1.amazonaws.com/wire-taco-test/custom_backend_staging.json
- Click LOG IN WITH SSO 
- You should see enter SSO dialog because there is no SSO code returned by the endpoint.
#### APK
[Download build #1369](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1369/artifact/build/artifact/wire-dev-PR2639-1369.apk)
